### PR TITLE
refactor(config-dotfiles): Update notes

### PR DESCRIPTION
**Pull request by axonasif**
## Description
Adds a few more important notes about dotfiles behavior

## Related Issue(s)

#1651 
@loujaybee had mentioned that we could document when dotfiles are installed.

## Release Notes

```release-note
```

## Documentation

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1631"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1631"></a>

### CODE REVIEW

File: gitpod/docs/config-dotfiles.md [hunk 1]
```diff
@@ -26,13 +26,17 @@ Gitpod will recognize and run one of the following install scripts from your dot
...
-Note: Your installation script will be terminated if it exceeds 120 seconds.
-
-Make sure to make your installation script executable with `chmod 755 <install-script-name>.sh` before committing and pushing to your dotfiles repository.
-
-If there is no install script, your dotfiles will be symlinked into `/home/gitpod`.
-
-The dotfiles repository installation logs are saved to `~/.dotfiles.logs`
+Important notes about dotfiles and install script:
+
+- Dotfiles installation is done before the [`before:`](/docs/config-start-tasks#prebuild-and-new-workspaces) task spec every time you launch a workspace, the IDE will not start until your install script exits or automatic files linking is done.
```
Comment (loujaybee) [inline gitpod/docs/config-dotfiles.md:31]: Could we mention the lifecycle of the backend (IDE) loading in here, also? @axonasif

Comment (axonasif) [reply gitpod/docs/config-dotfiles.md:31]: Yeah, I think it is better for our dotfiles users if we can provide more info like these. Can you give me some pointers(or suggest changes) please @loujaybee 🙏 ?

Comment (easyCZ) [reply gitpod/docs/config-dotfiles.md:31]: Seeing that this related to my original [FR](https://github.com/gitpod-io/website/issues/1651).

IMO, a diagram would speak a thousand words here. It makes it easy to visually grep and can provide a good high level feel for the cycle without having to read the fine print (but the user can always refer back to the details if needed).

Comment (axonasif) [reply gitpod/docs/config-dotfiles.md:31]: @Nancy-Chauhan can you please help us with a diagram here?

Comment (Siddhant-K-code) [reply gitpod/docs/config-dotfiles.md:31]: Maybe this diagram would be more easier to understand the workflow: 
image

// cc: @easyCZ @axonasif

**Comment by Cahllagerfeld** [comment]
Hey @loujaybee and @axonasif just a small ping, if the PR is still relevant, or if it can be closed :D

**Comment by Siddhant-K-code** [comment]
👋🏼 @loujaybee & @axonasif is this still relevant Or I can close this PR?

**Comment by Siddhant-K-code** [comment]
Looks like a stale PR. Closing it.

**Pull request closed without merging**